apps/etterlatte-saksbehandling-ui/client/src/components/behandling/sjekkliste/Sjekkliste.tsx

```diff
@@ -47,6 +48,11 @@ export const Sjekkliste = (props: { behandling: IBehandlingReducer }) => {
 
   const configContext = useContext(ConfigContext)
 
+  const rutinerBP =
+    'https://navno.sharepoint.com/:w:/r/sites/Pilot-Doffen/Delte dokumenter/General/Rutiner for sjekklister i Gjenny/BP - Rutine sjekkliste førstegangsbehandling.docx?web=1'
+  const rutinerOMS =
+    'https://navno.sharepoint.com/:w:/r/sites/Pilot-Doffen/Delte dokumenter/General/Rutiner for sjekklister i Gjenny/OMS - Rutine sjekkliste førstegangsbehandling .docx?web=1'
```

Comment:
Ser det er et lite mellomrom mellom `førstegangsbehandling` og `.docx`. Har ikke tilgang til dokumentet, så jeg får ikke sjekket om det er gyldig lenke eller ei

Reply:
Dette er forøvrig litt midlertidige rutinebeskrivelser.